src/driver/multistage.hpp

```diff
@@ -25,33 +26,37 @@
 
 namespace parthenon {
 
-struct Integrator {
-  Integrator() = default;
-  Integrator(int nstages, std::vector<Real> beta) : nstages(nstages), beta(beta) {}
+struct StagedIntegrator {
+  StagedIntegrator() = default;
+  StagedIntegrator(int nstages, std::vector<Real> beta) : nstages(nstages), beta(beta) {}
```

Comment:
Make beta const reference to avoid a copy.

Reply:
just deleted this unused constructor